CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 
+# 2.11.2 (2018-08-07)
+- Addressed a Windows compatibility issue with `skyux-builder-plugin-stache` version `1.5.0`, updated to version `1.5.1`.
```

Comment:
This doesn't read very well, and we should have some mention of the issue.  Might not hurt to just link to the plugin change log if you don't want to call out the issue here.
```
- Updated the [skyux builder plugin stache](https://github.com/blackbaud/skyux-builder-plugin-stache/blob/master/CHANGELOG.md) to `1.5.1`.